src/overlays/actors/ovl_Dm_Ravine/z_dm_ravine.h

```diff
@@ -7,7 +7,10 @@ struct DmRavine;
 
 typedef struct DmRavine {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x8];
+    /* 0x144 */ void (*actionFunc)(Actor*, GlobalContext*); // unused
```

Comment:
Do `typedef void (*DmRavineActionFunc)(struct EnTest3*, struct GlobalContext*);` instead, and maybe make a comment like `// Although this is unused, it is set to the actor's destroy function for some reason`, which is what appears to be happening which is super weird lol